source/guides/guides/parallelization.md

```diff
@@ -2,45 +2,74 @@
 title: Parallelization
 ---
 
-{% note warning %}
-This documentation is still a work in progress.
+{% note info %}
+
+# {% fa fa-graduation-cap %} What You'll Learn
+
+- How to parallelize test runs
+- How to group test runs
+- Strategies for grouping test runs
+- How test spec load-balancing works
```

Comment:
Should this be "How spec load-balancing..."?

Reply:
This is a little personal dilemma I have. Sometimes I just say "spec" and other times I say "test spec," because I feel like some people get confused with just saying "spec." For example, we use `coo.spec.js` for our filename conventions, but a lot of people just do `cool.test.js`. This is actually the default from people with Jest experience. So Idk, whatever it is, I want to be consistent. @jennifer-shehane thoughts?

Reply:
[Here's the steer](https://github.com/cypress-io/cypress-documentation/pull/829#pullrequestreview-145659913) from @chrisbreiding :

>[We] try to refer to 'tests' as the individual tests within a file (`it` blocks) and 'specs' as test files.